wercker.yml

```diff
@@ -9,16 +9,16 @@ build:
         code: ./build
 
     - script:
-        name: copy binary
-        code: cp bin/linux_amd64/farva-gateway "$WERCKER_OUTPUT_DIR"
+        name: copy binaries
+        code: cp bin/linux_amd64/* "$WERCKER_OUTPUT_DIR"
 
 deploy:
     box: nginx:1.10.0
     steps:
     - internal/docker-push:
         username: $QUAY_USERNAME
         password: $QUAY_PASSWORD
-        cmd: /pipeline/source/farva-gateway
+        working-dir: /pipeline/source/
```

Comment:
The docker container built by wercker previously ran `farva`. With `farva-monitor` in the same container, do we now launch the container with different cmds explicitly provided?